iOS/Flows/Conversation/Cells/MessageSubcell.swift

```diff
@@ -69,8 +69,11 @@ class MessageSubcell: UICollectionViewCell {
 
         self.detailView.height = MessageDetailView.height
         self.detailView.alpha = messageLayoutAttributes.detailAlpha
+
+        self.content.isUserInteractionEnabled = messageLayoutAttributes.detailAlpha == 1
```

Comment:
Simply enable or disable the user interaction based on whether detail alpha is 1. This should get us the right context menu behavior.